bundle/schema/schema_test.go

```diff
@@ -14,7 +14,15 @@ func TestIntSchema(t *testing.T) {
 
 	expected :=
 		`{
-			"type": "number"
+			"anyOf": [
```

Comment:
The current approach is not ideal, and there is indeed room for refactoring out the "Go struct" -> "JSON Schema" transformation from bundle-specific transformations. That being said, I think this approach is fine given we don't anticipate a lot of changes to `bundle schema` generation, and it'll not be that hard to refactor if we need to in the future.